waku/v2/node/config.nim

```diff
@@ -458,11 +459,13 @@ proc parseCmdArg*(T: type Port, p: TaintedString): T =
 proc completeCmdArg*(T: type Port, val: TaintedString): seq[string] =
   return @[]
 
-func defaultListenAddress*(conf: WakuNodeConf): ValidIpAddress =
```

Comment:
Can we continue to use `func` here? 
context: https://status-im.github.io/nim-style-guide/language.proc.html